protocol/urls/urls_test.go

```diff
@@ -1,9 +1,8 @@
 package urls
 
 import (
-	"testing"
-
 	"github.com/stretchr/testify/require"
+	"testing"
```

Comment:
the linter is going to be unhappy about this one :)

Reply:
solved by using `goimports -w -local 'github.com/ethereum/go-ethereum,github.com/status-im/status-go,github.com/status-im/markdown' {filename}`